pkgs/applications/misc/sioyek/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gumbo
+, harfbuzz
+, jbig2dec
+, mupdf
+, openjpeg
+, qt3d
+, qtbase
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sioyek";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "ahrm";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-UhZuELWfFfqj1jjCgZTD+X44i7jMpeDTbClkZarV2Zw=";
+  };
+
+  buildInputs = [ gumbo harfbuzz jbig2dec mupdf openjpeg qt3d qtbase ];
+
+  nativeBuildInputs = [ wrapQtAppsHook ];
+
+  buildPhase = ''
+    # Remove nonexistent lib and insert missing ones
+    sed -i 's/-lmupdf-threads/-lfreetype -lgumbo -ljbig2dec -lopenjp2 -ljpeg/' pdf_viewer_build_config.pro
+    qmake pdf_viewer_build_config.pro
+    make
+  '';
+
+  installPhase = ''
+    install -Dm755 sioyek $out/bin/sioyek
+    install -Dm644 tutorial.pdf $out/bin/tutorial.pdf
+    install -Dm644 -t $out/bin/ pdf_viewer/{keys,prefs}.config
```

Comment:
maybe $out/share?

Reply:
Again, I'm not sure about the search paths. These configs are the default configs, and they are loaded by default. If they are not found then keybinds, etc will all not be loaded at all and you can't interact with the program.

Reply:
Unfortunately I'm not sure how the paths work in this program, but it defaults to `/bin`. In any case these are the default configs, not the user configs, but they do still need to be loaded.

```
default_config_path: /nix/store/54kvg20fhmf7iq5091vddc3ksg062vq6-sioyek-1.1.0/bin/prefs.config
Error: default_config_path: /nix/store/54kvg20fhmf7iq5091vddc3ksg062vq6-sioyek-1.1.0/bin/prefs.config doesn't exist!
default_keys_path: /nix/store/54kvg20fhmf7iq5091vddc3ksg062vq6-sioyek-1.1.0/bin/keys.config
Error: default_keys_path: /nix/store/54kvg20fhmf7iq5091vddc3ksg062vq6-sioyek-1.1.0/bin/keys.config doesn't exist!
```